### PR TITLE
[SIMS-#900][Forms]Relationship-status rename

### PR DESCRIPTION
**Pull request by guru-aot**
- Relationship status for marriedCommonLaw is renamed to married and otherSingle is renamed to other in all the workflow and the student application forms for the 3 years. This has been done to be in consistent with our code relationshipstatus.
- dependantStatus hidden variable event trigger has been updated to AnyChange to hide and show the partnerInformation tab, which was inconsistent. Also moved to the appropriate screen.

### CODE REVIEW

File: sources/packages/forms/src/form-definitions/sfaa2021-22.json [hunk 1]
```diff
@@ -13135,7 +13334,82 @@
...
+                    "customClass": "",
+                    "modalEdit": false,
+                    "defaultValue": null,
+                    "persistent": true,
+                    "protected": false,
+                    "dbIndex": false,
+                    "encrypted": false,
+                    "redrawOn": "data",
+                    "customDefaultValue": "",
+                    "calculateValue": "//alert (data.dependantstatus);\nif (data.hasDependents === 'no' && (data.relationshipStatus === 'single' || data.relationshipStatus === 'other') && data.outOfHighSchoolFor4Years === 'no' && data.howWillYouBeAttendingTheProgram === \"Full Time\") {\n  instance.setValue ('dependant');\n}\nelse {\n  instance.setValue ('independant');\n}",
```
Comment (andrewsignori-aot) [inline sources/packages/forms/src/form-definitions/sfaa2021-22.json]: Please remove the commented alert.

**Review by andrewsignori-aot — COMMENTED** [review]
Nice work! Now the data to be send to ESDC will be correct for e-cert 👍 
Can you please ensure that below option is checked?
image

**Review by dheepak-aot — APPROVED** [review]
LGTM @guru-aot 👍

**Review by henryoforeh-aot — APPROVED** [review]

**Review by ann-aot — APPROVED** [review]
👍

**Review by andrewsignori-aot — APPROVED** [review]
Looks good 👍

**Review by abschwenker — APPROVED** [review]

**Pull request merged**